gnome-initial-setup/gnome-initial-setup-copy-worker.c

```diff
@@ -2,12 +2,14 @@
 
 /* Copies settings installed from gnome-initial-setup and
  * sticks them in the user's profile */
+#include "config.h"
 
 #include <pwd.h>
 #include <string.h>
+#include <glib/gstdio.h>
 #include <gio/gio.h>
-#include <gnome-keyring.h>
```

Comment:
From a quick `git grep`: `gnome-initial-setup/meson.build` appears to have a `dependency ('gnome-keyring-1')` which might need to be dropped?